frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx

```diff
@@ -54,7 +61,13 @@ export const MoleculeSelector: React.FC<MoleculeSelectorProps> = ({
           {...params}
           variant="standard"
           fullWidth
-          label={isGeisa ? "GEISA 2020 Molecule" : "HITRAN 2020 Molecule"}
+          label={
+            isHitemp
+              ? "Molecule"
+              : isGeisa
+              ? "GEISA 2020 Molecule"
+              : "HITRAN 2020 Molecule"
+          }
```

Comment:
just use "Molecule" ? Simpler, clearer and no conditions needed.

Reply:
it is perfect I think @erwanp